p2p/message.go

```diff
@@ -124,58 +113,63 @@ var dataExcludedMsgs = map[uint64]struct{}{
 	0x10: empty, //Receipts
 }
 
-func SendEthSubproto(w MsgWriter, msgcode uint64, data interface{}, peers ...discover.NodeID) error {
+func SendEthSubproto(w MsgWriter, msgcode uint64, data interface{}, connInfoCtx ...interface{}) error {
 	size, r, err := rlp.EncodeToReader(data)
 
 	if err != nil {
 		return err
 	}
 
-	var peer discover.NodeID
-	if len(peers) == 1 {
-		peer = peers[0]
+	err = w.WriteMsg(Msg{Code: msgcode, Size: uint32(size), Payload: r})
+	currentTime := time.Now()
+	msgType, ok := ethCodeToString[msgcode]
+	if !ok {
+		msgType = fmt.Sprintf("UNKNOWN_%v", msgcode)
 	}
-
-	msgType := ethCodeToString[msgcode]
-	if _, ok := dataExcludedMsgs[msgcode]; ok {
-		data = "<OMITTED>"
+	obj := data
+	if obj, ok = dataExcludedMsgs[msgcode]; ok {
```

Comment:
Suggestion for debugging the size=1 issue. Try adding another condition when determining whether to omit the data or not. For example, to log data that's less than size 10:
`if obj, ok = dataExcludedMsgs[msgcode]; ok && size < 10 {`